website/content/docs/connect/gateways/api-gateway/configuration/gatewaypolicy.mdx

```diff
@@ -0,0 +1,258 @@
+---
+layout: docs
+page_title: GatewayPolicy configuration reference
+description: Learn about the configuration options for the GatewayPolicy configuration resource. GatewayPolicy resources define API gateway polcies for Consul service mesh on Kubernetes.
+---
+
+# GatewayPolicy
+
+This topic provides reference information for the API gateway policy resource, which defines API gateway policies for Consul service mesh on Kubernetes-orchestrated networks.
+
+## Configuration model
+
+The following list outlines field hierarchy, data types, and requirements in a gateway policy resource. Click on a property name to view additional details, including default values.
+
+- [`apiVersion`](#apiversion): string | required | must be set to `gateway.networking.k8s.io/v1beta1`
+- [`kind`](#kind): string | required | must be set to `GatewayPolicy`
+- [`metadata`](#metadata): map | required
+   - [`name`](#metadata-name): string | required 
+   - [`namespace`](#metadata-namespace): string  | `default`
+- [`spec`](#spec): map | required
+   - [`targetRef`](#spec-targetref): map | required 
+      - [`name`](#spec-targetref): string | required 
+      - [`kind`](#spec-targetref): string | required | must be set to `Gateway`
+      - [`group`](#spec-targetref): string | required | must be set to `gateway.networking.kuberenetes.io`
+      - [`sectionName`](#spec-targetref): string 
+   - [`override`](#spec-override): map | required 
+      - [`jwt`](#spec-override-jwt): map | required 
+         - [`providers`](#spec-override-providers): list | required 
+            - [`name`](#spec-override-providers): string | required 
+            - [`verifyClaims`](#spec-override-providers): map | required 
+               - [`path`](#spec-override-providers): list of strings | required 
+               - [`value`](#spec-override-providers): string | required 
+   - [`default`](#spec-default): map | required 
+      - [`jwt`](#spec-default-jwt): map | required 
+         - [`providers`](#spec-default-providers): list | required 
+            - [`name`](#spec-default-providers): string | required 
+            - [`verifyClaims`](#spec-default-providers): map | required 
+               - [`path`](#spec-default-providers): list of strings | required 
+               - [`value`](#spec-default-providers): string | required 
+
+## Complete configuration
+
+When every field is defined, a gateway policy has the following form: 
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: GatewayPolicy
+metadata:
+   name: <name for the policy>
+   namespace: <Consul namespace the policy applies to>
+spec:
+  targetRef:
+    name: gateway
+    kind: Gateway
+    group: gateway.networking.kuberenetes.io
+    sectionName: <name of a specific section the policy applies>
+  override:
+    jwt:
+      providers:
+      - name: "<name of the provider all listeners verify against>"
+        verifyClaims:
+        - path:
+          - "<>"
```

Comment:
I think this should be `"aud"` based on your example above

Reply:
I want to use a general description in this field similar to the values in the `name` fields, but I don't know what that text is. Using `aud` seems too specific

Reply:
Ah, OK, thanks. I think I understand. The `api.apps . . .` value is the expected value of an `aud` claim, so it should match that type. The value `user` matches the claim types in the path in the default configuration in this snippet.